### PR TITLE
Chore/refactor

### PR DESCRIPTION
**Pull request by alexsavio**

### CODE REVIEW

**Comment by alexsavio** [comment]
about the authors, could you use https for mine and http for ovv.wtf

File: roles/python/tasks/main.yml [hunk 1]
```diff
@@ -2,16 +2,14 @@
...
+      - python3-dev
+      - python3-pip
 
 - name: Update pip and setuptools
   pip:
     name: '{{ item }}'
-    state: forcereinstall
+    state: latest
   with_items:
     - pip
```
Comment (webknjaz) [inline roles/python/tasks/main.yml]: Hopefully, next version of pip module will support multiple entries in `name` :)
Probably coming in Ansible 2.7. My intern/trainee is currently working on this.

File: roles/python/tasks/main.yml [hunk 2]
```diff
@@ -2,16 +2,14 @@
 
 - name: Install some apt packages
   apt:
-    name: '{{ packages }}'
-  vars:
-    packages:
-    - python-dev
-    - python-pip
+    name:
+      - python3-dev
```
Comment (webknjaz) [inline roles/python/tasks/main.yml]: FWIW I prefer unindented sequence.

File: roles/docker-compose/tasks/install_docker_compose.yml [hunk 3]
```diff
@@ -2,30 +2,18 @@
...
-    name: '{{ item }}'
-    state: forcereinstall
-  with_items:
-    - pipsi
-  executable: pip3
   become: yes
   become_method: sudo
 
 - name: Install docker-compose
   command:
```
Comment (webknjaz) [inline roles/docker-compose/tasks/install_docker_compose.yml]: Replace this with:
```python
pip:
  name: docker-compose
  version: 1.22.0
```
(rm `args`)

File: AUTHORS.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
+
+Contributors
+------------
+
+Alexandre Savio <http://alexsavio.github.io/>
+Artur Czepiel <http://artcz.pl>
+@bamartos
+Hardion Vincent <https://github.com/hardion>
+Mark Smith <@judy2k>
+Quentin Dawans <ttps://ovv.wtf>
```
Comment (ovv) [inline AUTHORS.md]: Missing the `h` :)

Comment (webknjaz) [reply AUTHORS.md]: Where?

Comment (ovv) [reply AUTHORS.md]: Quentin Dawans <https://ovv.wtf>

Comment (webknjaz) [reply AUTHORS.md]: oh, I completely missed that :D

File: roles/python/tasks/main.yml (hunk 1 above)
Comment (alexsavio) [reply roles/python/tasks/main.yml]: I will leave it like this for now. thanks :)

File: roles/python/tasks/main.yml (hunk 2 above)
Comment (alexsavio) [reply roles/python/tasks/main.yml]: Ok, fixed. :)

File: AUTHORS.md [hunk 5]
```diff
@@ -0,0 +1,11 @@
+
+Contributors
+------------
+
+Alexandre Savio <http://alexsavio.github.io/>
```
[the remark this answers is not in the document]
Comment (alexsavio) [reply AUTHORS.md]: Fixed

File: AUTHORS.md (hunk 4 above)
Comment (alexsavio) [reply AUTHORS.md]: Me too :) fixed

**Pull request merged**